fasta/multiFa.go

```diff
@@ -22,6 +23,19 @@ func RefPosToAlnPos(record Fasta, RefPos int) int {
 	return AlnPos
 }
 
+//RefPosToAlnPosCounter is like RefPosToAlnPos, but the starting AlnPos and refCounter can be set.
```

Comment:
I think a programmer might need more explanation for what to pass in as refStart and alnStart